src/main/java/decodex/parser/Parser.java

```diff
@@ -57,27 +57,29 @@ public String getCommandType(String userInput) throws ParserException {
         return commandType;
     }
 
-    /**
-     * Returns the Argument portion of the user input as a whole string.
-     *
-     * @param userInput The input specified by the user.
-     * @return The argument portion of the user input as an array.
-     */
-    public String getUserArgument(String userInput) throws ParserException {
+    public String getModuleName(String userInput) throws ParserException {

```

Comment:
Perhaps add some JavaDoc comments?

Reply:
JavaDocs can be done in a separate issue/PR as it is not the intended change in this PR

Reply:
Even though it was not the intended change, code that were introduced should still be of a certain standard (e.g comments, code quality) before being merged in.